lib/config/definitions.ts

```diff
@@ -7,6 +7,13 @@ import * as pep440Versioning from '../versioning/pep440';
 import type { RenovateOptions } from './types';
 
 const options: RenovateOptions[] = [
+  {
+    name: 'allowGitLfs',
+    description: 'If true allow git-lfs to pull files.',
+    type: 'boolean',
+    default: false,
```

Comment:
Per your comment, I think it's better to default this to `true` for now, unless you want to hold it until the next breaking release.